### PR TITLE
Merging Datasets and video_rgb.py

### PR DESCRIPTION
**Pull request by yahya010**
First, downloading different datasets with v2d will result in the same filenames for each dataset, e.g. downloading HowTo100M with v2d will result in files named
data/raw/howto100m/v2d
| --- 0000000000.tar
| --- 0000000001.tar
...
| --- 0000006000.tar
and downloading hd-vila with v2d will also result in files named something like
data/raw/hdvila/v2d
| --- 0000000000.tar
| --- 0000000001.tar
...
| --- 0000002020.tar
so then we need to make sure when we merge them (a) that there's metadata within the tar files which says which dataset a tar file came from and (b) that the names of the files don't overlap. So in this example would want
data/4m/video_rgb/
| --- 0000000000.tar
| --- 0000000001.tar
...
| --- 0000008020.tar
where the first 6000 are from howto100m and the second 2020 are from hdvila.

For video_rgb.py, still needs further testing when todi is back

### CODE REVIEW

**Comment by markus583** [comment]
Thanks for this! Merging looks fine to me. But I am not sure about the options for `video_rgb.py`. I thought we do not want to downsample and crop at all in this stage but rather leave the videos raw?

**Review by markus583 — APPROVED** [review]
Looking good, thanks!

File: fourm/data/merge_data.sh [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+    tar -rf "$file" metadata.txt
+    rm metadata.txt
+    
+    # Copy to output directory with new name
+    cp "$file" "$OUTPUT_DIR/$new_filename"
+    
+    ((counter++))
+done
+
+echo "Merging complete. Output files are in $OUTPUT_DIR"
```
Comment (kdu4108) [inline fourm/data/merge_data.sh:46]: Can you add the time thing so people know?

**Comment by kdu4108** [comment]
Looks great to me as well! However I agree with Markus; in fact, I don't think we need the python script at all actually? If it's not useful, I'd recommend deleting it from here entirely.

Also just noting -- as discussed, we will want both a `toy` and a `full` data directory
e.g.
`/store/swissai/a08/data/raw/toy/...` and `/store/swissai/a08/data/raw/full/...`.

I think we don't need to handle for this now since we're only working with toy directory, but just noting that when we move to downloading full datasets, we may need to change some filenames.

**Comment by kdu4108** [comment]
Feel free to 1) delete the python file 2) add the total time echo and then 3) merge!

**Pull request merged**